packages/auth/src/controllers/teams.ts

```diff
@@ -0,0 +1,26 @@
+import { Request, Response } from 'express'
+import { validateAll } from 'indicative/validator'
+
+export const createTeam = async (request: Request, response: Response) => {
+    const { manager, body, authUser } = request
+    const { name } = await validate(body)
+
+    const team = await manager('Team').create({
+        name,
+        customer_id: request.authUser!.id,
```

Comment:
Should not be `customer_id` . Should be based on whatever the user passed as the `customerResourceName`

So these controllers need to have access to the config passed from auth. 

So for example, if the user called the auth `User` then this would be `user_id`